README.md

```diff
@@ -269,7 +270,15 @@ Here is a sample configuration:
   }
 }
 ```
+### FhirVersion Specify
+| fhirVersion | Desciption |
+| ----- | ----- |
+|Stu3|Specify STU 3 version for the configuration file|
+|R4|Specify R4 version for the configuration file|
+|Empty or Null| The configuration file will be specified the same version with the running executable file.

```

Comment:
Add another line for other values?  Other values will raise an **exception.

Reply:
Thanks , added a new line in table.

Reply:
Is the new line already pushed into this PR? It seems I cannot find it.

Reply:
For empty or null, suggest changing the working to : "The configuration file targets the same FHIR version as the executable."

Reply:
Thanks, updated here.